tests/ci/docker_images/windows/build_images.ps1

```diff
@@ -1,6 +1,9 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0 OR ISC
 
-docker build -t aws-lc/windows_base:2019 .\windows_base
-docker build -t vs2015 .\vs2015
-docker build -t vs2017 .\vs2017
+. .\common.ps1
+
+Invoke-Command { docker build -t aws-lc/windows-2022:base .\windows-2022_base }
+Invoke-Command { docker build -t windows-2022:vs2022-sde .\windows-2022_vs2022-sde }
+Invoke-Command { docker build -t windows-2022:vs2017-sde .\windows-2022_vs2017-sde }
+Invoke-Command { docker build -t windows-2022:vs2015 .\windows-2022_vs2015 }
```

Comment:
Nit: Would be great if ordering could align with `tests/ci/docker_images/windows/push_images.ps1`.

Also, should we add the sde tag to these docker images? No strong opinions on my end.

Context: I added SDE tools since we could be looking to some docker image consolidation (`CryptoAlg-1841`). We're not actually running Windows SDE tests through this image though, we run the Windows SDE tests through Github Actions. I included the setup here since I had already investigated it (setting up Windows is a pain...) and it could be useful for future testing purposes."

Reply:
We're actually trying to migrate the Windows SDE tests to CodeBuild per CryptoAlg-3113. I mentioned this more clearly in the description/callouts

Reply:
> should we add the sde tag to these docker images? No strong opinions on my end.

The full tag for sde images are either `windows-2022_vs2022-sde` or `windows-2022_vs2019-sde`. Do you mean adding more tags or rename them in a way?